default.json

```diff
@@ -23,7 +20,8 @@
       "groupName": "openstack-k8s-operators",
       "matchPackagePatterns": ["^github.com/openstack-k8s-operators"],
       // force pseudo versioning
-      "versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
+      //"versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
+      "versioning": "regex:^(?<major>0)\\.(?<minor>1)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
```

Comment:
This probably needs to be ?<minor>3 as default.json is for the main branch

Reply:
I don't actually think we need any changes here. (this was in error). Latest update should remove it